libs/domain/checkout/src/services/default-checkout.service.ts

```
@@ -57,14 +56,7 @@ export class DefaultCheckoutService implements CheckoutService {
           validity.every(({ validity }) => validity === Validity.Valid)
         ),
         switchMap((canCheckout) =>
-          canCheckout
-            ? this.preparePayload()
-            : throwError(
-                () =>
-                  new Error(
-                    'Cannot checkout, check your cart value limits and validations'
-                  )
-              )
+          canCheckout ? this.preparePayload() : of()
```

not sure if we should fail silently? 
Throwing an error gives ability to decide in the component, what to do with it. 

But I agree, that the error message here is too specific.

---

I noticed the error is thrown when expected; when some of the forms are not valid, we have form validation and this error seems too much.

---

` canCheckout ? this.preparePayload() : of(undefined)`